[PATCH] TOMOYO: Fix refcount leak in tomoyo_mount_acl().

commit db5ca356d8af8e43832c185ceec90850ff2ebb45 in security-testing-2.6.git. 

In tomoyo_mount_acl() since 2.6.36, reference to device file (e.g. /dev/sda1)
was leaking.

Signed-off-by: Tetsuo Handa <penguin-kernel@I-love.SAKURA.ne.jp>
---
 security/tomoyo/mount.c |    1 +
 1 file changed, 1 insertion(+)

--- linux-2.6.36.4.orig/security/tomoyo/mount.c
+++ linux-2.6.36.4/security/tomoyo/mount.c
@@ -143,6 +143,7 @@ static int tomoyo_mount_acl(struct tomoy
 			goto out;
 		}
 		requested_dev_name = tomoyo_realpath_from_path(&path);
+		path_put(&path);
 		if (!requested_dev_name) {
 			error = -ENOENT;
 			goto out;
